### ISSUE TITLE
Node 6.0.0 compatibility warning

### ISSUE THREAD

**Issue by lee-houghton** [astro/node-expat#150]
I get this warning when using `node-expat` with Node 6.0.0. I've noticed it with a few other packages but I'm not sure if it's an issue with Nan or this library. I'm using `nan 2.3.0` and `node-expat 2.3.13`. I couldn't see anything obvious in the C++ code but it's been a few major versions since I've been writing binary modules for node.

```
(node) v8::ObjectTemplate::Set() with non-primitive values is deprecated
(node) and will stop working in the next major release.

==== JS stack trace =========================================

Security context: 000003147AAC9FC9 <JS Object>#0#
    1: .node [module.js:568] [pc=000001FC9EA35944] (this=00000149056FDD01 <an Object with map 000003E8E5834A39>#1#,module=000002289523E8B1 <a Module with map 000003E8E5817979>#2#,filename=000002289523E889 <String[70]: C:\Somewhere\node_modules\node-expat\build\Release\node_expat.node>)
    2: load [C:\Somewhere\node_modules\coffee-script\lib\coffee-script\register.js:45] [pc=000001FC9E99E634] (this=000002289523E8B1 <a Module with map 000003E8E5817979>#2#,filename=000002289523E889 <String[70]: C:\Somewhere\node_modules\node-expat\build\Release\node_expat.node>)
    3: tryModuleLoad(aka tryModuleLoad) [module.js:415] [pc=000001FC9E93C21D] (this=000003147AA04189 <undefined>,module=000002289523E8B1 <a Module with map 000003E8E5817979>#2#,filename=000002289523E889 <String[70]: C:\Somewhere\node_modules\node-expat\build\Release\node_expat.node>)
    4: _load [module.js:407] [pc=000001FC9E937662] (this=00000149056FDDD1 <JS Function Module (SharedFunctionInfo 000001490562A0C1)>#3#,request=000002289523D7D9 <String[70]: C:\Somewhere\node_modules\node-expat\build\Release\node_expat.node>,parent=0000022895222239 <a Module with map 000003E8E5817979>#4#,isMain=000003147AA04299 <false>)
    5: require [module.js:~463] [pc=000001FC9EA1920B] (this=0000022895222239 <a Module with map 000003E8E5817979>#4#,path=000002289523D7D9 <String[70]: C:\Somewhere\node_modules\node-expat\build\Release\node_expat.node>)
    6: require(aka require) [internal/module.js:20] [pc=000001FC9E9472E6] (this=000003147AA04189 <undefined>,path=000002289523D7D9 <String[70]: C:\Somewhere\node_modules\node-expat\build\Release\node_expat.node>)
    7: bindings [C:\Somewhere\node_modules\bindings\bindings.js:76] [pc=000001FC9EA32616] (this=000003147AAE81B9 <JS Global Object>#5#,opts=0000020CCF6E4419 <String[10]: node_expat>)
    8: /* anonymous */ [C:\Somewhere\node_modules\node-expat\lib\node-expat.js:4] [pc=000001FC9EA30ECC] (this=0000022895214751 <an Object with map 000001477C107B71>#6#,exports=0000022895214751 <an Object with map 000001477C107B71>#6#,require=00000228952171E1 <JS Function require (SharedFunctionInfo 000001490565BBF9)>#7#,module=0000022895214701 <a Module with map 000003E8E5817979>#8#,__filename=00000228952146D9 <String[58]: C:\Somewhere\node_modules\node-expat\lib\node-expat.js>,__dirname=0000022895217179 <String[44]: C:\Somewhere\node_modules\node-expat\lib>)
```

**Comment by noisygerman** [comment]
Looks like we need to change the minimum version of the nan package to 2.3.1 - I'll check it out (see https://github.com/Automattic/node-canvas/issues/758)

**Comment by noisygerman** [comment]
This seems to have been resolved with Node 6.1

**Comment by astro** [comment]
Thank you for the information.
